src/main/java/seedu/address/model/property/FilterPropsByCharacteristicsPredicate.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.property;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.characteristics.Characteristics;
+
+/**
+ * Tests that a given {@code Property}'s {@code DesiredCharacteristics} contains the given characteristic.
+ */
+public class FilterPropsByCharacteristicsPredicate extends AbstractFilterPropsPredicate {
+
+    private static final String CHARACTERISTIC_DELIMITER = ";";
```

Comment:
```suggestion

```
Redundant line of code!

Reply:
Removed in FilterBuyersByCharacteristicsPredicate as well